src/style.css

```diff
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: book;
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: heavy;
+  font-weight: bold;
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
+}
+
+.SearchBar {
+  position: relative;
+  margin-top: 20px;
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  display: block;
+  width: 19px;
+  height: 19px;
+}
+
+.search_box {
```

Comment:
```suggestion
.search_box {
  outline: none;
```
аутлайн скидаємо для інпутів

Reply:
<img width="326" alt="image" src="https://user-images.githubusercontent.com/66749919/199314397-c03bb270-bbfe-46ce-a9c5-220ef91aaaa9.png">